components/common/BackToHome.tsx

```diff
@@ -4,8 +4,8 @@ import Link from 'next/link';
 const BackToHome = ({ className = 'my-12 max-w-4xl mx-auto px-4' }: { className?: string }): JSX.Element => {
   return (
     <div className={className}>
-      <Link href="/">
-        <a className="no-underline hover:underline">← Back to home</a>
+      <Link href="/" className="no-underline hover:underline">
+        ← Back to home
```

Comment:
also, I think we should redirect user to last visited page in the stack, not directly to Home.

If one goes to "All Issues" page, then open an issue, & then click this - it redirects back to home. not good UX.

let's take it up separately. I'll open an issue to remember.